pages/linux/tracepath.md

```diff
@@ -0,0 +1,28 @@
+# tracepath
+
+> Trace the path to a network host discovering MTU along this path.
+> More information: <https://manned.org/tracepath>.
+
+- A preferred way to trace the path to a host:
+
+`tracepath -p 33434 {{host}}`
```

Comment:
```suggestion
`tracepath -p {{33434}} {{host}}`
```

Reply:
I have read https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#page-structure where this is mentioned.

Is there some other publicly available discussion about the {{handlebars}} subject, please? Since there is no very specific keyword associated, it is hard to search it.

Reply:
Not that I'm aware of. Maybe @bl-ue or @waldyrious know of something

Reply:
We call it [token syntax](https://github.com/tldr-pages/tldr/blob/a108e73ce3cff3500e106f172db011a66dcfa4a5/contributing-guides/style-guide.md#token-syntax), and we usually put it on any value that the user has a choice in.